apps/api/src/app/testing/auth.controller.ts

```diff
@@ -1,8 +1,12 @@
 import { Controller, Get, UseGuards } from '@nestjs/common';
 import { UserAuthGuard } from '../auth/framework/user.auth.guard';
 import { ExternalApiAccessible } from '../auth/framework/external-api.decorator';
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { ApiKeyAuth } from '../shared/framework/swagger/api.key.security';
 
 @Controller('/test-auth')
+@ApiBearerAuth()
+@ApiKeyAuth()
 export class TestApiAuthController {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** I guess we can exclude this

Reply:
@rifont same here